### ISSUE TITLE
Outlook on the web: Accessibility: VoiceOver/NVDA cannot use keyboard input (tab) to enter the add-in

### ISSUE THREAD

**Issue by hello-chenchen** [OfficeDev/office-js#913]
accessibility software(VoiceOver and nvda) can't press "tab" and enter into the add-on right side panel.
And you can try to remove the **tabindex="0"** which is on the top of the add-on iframe panel div.
like below:
![image](https://user-images.githubusercontent.com/1878089/71237629-04479a00-233d-11ea-969b-74b0d5b6358a.png)

remove this one, and both VoiceOver and nvda can be worked.
like below:
![image](https://user-images.githubusercontent.com/1878089/71238052-183fcb80-233e-11ea-95b4-99dc427985c9.png)

But, this is office365 code, not my. So i can't do anything to fix that.
So, please check this one.

**Comment by lindalu-MSFT** [comment]
@hello-chenchen Thanks for bringing this accessibility issue to our attention! I assigned this to the Outlook Dev team for further investigation. Please be aware that it is a holiday week here at Microsoft and many people are out of office. It might be a few days before you get a response. I also assigned myself to follow up and be sure this is attended to. Cheers, happy holidays!

**Comment by exextoc** [comment]
We are able to repro this issue on outlook web client. Thanks for reporting this issue regarding. It has been put on our backlog. We unfortunately have no timelines to share at this point.

**Comment by rajeshk7** [comment]
I am able to get to the extension with tab. I am able to get to the cross button. Are you not able to reach to the cross button even ? Or you want to use tab button inside OneNote extension ?

**Comment by hello-chenchen** [comment]
@rajeshk7 , yes, as a blind user, they will need use tab button inside the add-ins.

**Comment by exextoc** [comment]
We looked at this and it is an add-in bug. Its the responsibility of an add-in to provide accessibility. We have reported it to OneNote add-in team and they will look into it.
